### PR TITLE
personal_data_added

### PR DESCRIPTION
**Pull request by Arpaninformationtech**
I have added my dada...

### CODE REVIEW

**Comment by arnabsen1729** [comment]
Hi Arpan, thanks for creating the PR. But don't make changes in the `.all-contributorsrc` instead, create a new file inside the `contributors` directory.

The filename should be `<your-github-username>.md`, and then add a small bio inside the file. 

**How to do so?**

- First remove the change that you made in the  `.all-contributorsrc` file
- Then create a new file and make a commit
- Simply push the commit and this PR will automatically be updated. 

If you still face any difficulty contact me on the whatsapp group.

File: .all-contributorsrc [hunk 1]
```diff
@@ -14,6 +14,7 @@
         "code"
       ]
     },
+
```
Comment (arnabsen1729) [inline .all-contributorsrc:17]: @Arpaninformationtech there is this extra line over here. Just remove this too.

Comment (Arpaninformationtech) [reply .all-contributorsrc]: ok done @arnabsen1729

**Comment by arnabsen1729** [comment]
@all-contributors add @Arpaninformationtech for code

**Pull request merged**